app/src/main/java/com/github/se/eventradar/model/repository/user/MockUserRepository.kt

```diff
@@ -110,13 +110,18 @@ class MockUserRepository : IUserRepository {
     }
   }
 
+  override suspend fun generateQRCode(userId: String): Resource<String> {
```

Comment:
I would imagine you would want a failure case for testing as well. maybe if the user is not in the user database, return a failure.